### PR TITLE
Remove ubuntu-1810 from spread.yaml

### PR DESCRIPTION
**Pull request by wmww**
`ubuntu-os-cloud/ubuntu-1910` [doesn't seem to be a thing](https://travis-ci.org/MirServer/mir/jobs/564109665)

### CODE REVIEW

File: spread.yaml [hunk 1]
```diff
@@ -22,8 +22,6 @@ backends:
                 image: ubuntu-os-cloud/ubuntu-1804-lts
             - ubuntu-19.04-64:
                 image: ubuntu-os-cloud/ubuntu-1904
-            - ubuntu-devel-64:
-                image: ubuntu-os-cloud/ubuntu-1810
```
Comment (Saviq) [inline spread.yaml]: This is actually a typo. This should say `1910` instead.

File: .travis.yml [hunk 2]
```diff
@@ -43,7 +43,6 @@ env:
   - DEBEMAIL="mir-ci-bot@canonical.com"
   - SPREAD_PATH=/tmp/spread
   matrix:
-  - SYSTEM=google:ubuntu-devel-64 VARIANT=amd64
```
Comment (Saviq) [inline .travis.yml]: This should remain.

File: spread.yaml (hunk 1 above)
Comment (wmww) [reply spread.yaml]: I tried that and it didn't work: https://travis-ci.org/MirServer/mir/jobs/564109665

```
2019-07-26 17:10:09 Allocating google:ubuntu-devel-64...
2019-07-26 17:10:09 Cannot allocate google:ubuntu-devel-64: cannot find any Google image matching "ubuntu-os-cloud/ubuntu-1910" on project "ubuntu-os-cloud"
2019-07-26 17:10:09 Successful tasks: 0
2019-07-26 17:10:09 Aborted tasks: 1
error: unsuccessful run
```

Comment (Saviq) [reply spread.yaml]: Make this `1904` then, it will get upgraded to devel in CI.

**Pull request merged**